### ISSUE TITLE
No license in the docs repo

### ISSUE THREAD

**Issue by vmbrasseur** [gazebosim/docs#556]
## Environment

N/A

## Description

This repository doesn't seem to have a license in it. The ROS2 docs are CC-BY, so I expect this is the license that'll be used for the Gazebo docs as well?

## Steps to reproduce

N/A

## Output
 
N/A

**Comment by vmbrasseur** [comment]
Hey there, team. Happy to send over a PR to add a license to the repo, if you'd like.

**Comment by azeey** [comment]
Yes, we should use CC-BY as the ROS 2 docs. A PR would be much appreciated!
